### DISCUSSION TITLE
Can Tasmoto support more modbus devices on one RS485 ?

### DISCUSSION THREAD

**Discussion by Rojer7084** [arendst/Tasmota#14996] [Q&A] [answered]
Dear tasmota member , 
  Tasmota quite powerfull framwork on IOT  but We need to know if we want to use more modbus device  such as power meter  , remote IO  on the same RS485 ( Modebus RTU Protocol ) , is possible or note  , if  possible how can we  develope them   

Many thanks for your advice.
Rojer

**Comment by gemu2015** [comment, marked as answer]
https://tasmota.github.io/docs/Smart-Meter-Interface/

there are some examples of multiple MODBUS devices on the same bus
